vulture/core.py

```diff
@@ -255,6 +255,17 @@ def by_size(item):
         return sorted(confidently_unused,
                       key=by_size if sort_by_size else by_name)
 
+    def make_whitelist(self, min_confidence=0, sort_by_size=False):
+        for item in self.get_unused_code(
+                min_confidence=min_confidence, sort_by_size=sort_by_size):
+            self.found_dead_code_or_error = True
+            if item.typ in ('unreachable_code', 'import'):
```

Comment:
Let's explicitly list the types that are supported here instead of listing the ones that aren't.